modules/documents/app/views/documents/index.html.erb

```diff
@@ -59,7 +59,7 @@ See COPYRIGHT and LICENSE files for more details.
 <% end %>
 
 <% @grouped.keys.sort.each do |group| %>
-  <%= augmented_collapsible_section(title: group.name) do %>
+  <%= augmented_collapsible_section(title: group.try(:name) || group.to_s) do %>
```

Comment:
On which object do we expect to call `name`?

I am asking because `User` defines a `name` method`, but also the `User`'s `to_s` method is effectively an alias of `name`.

So I still believe that this can be simplified to

```suggestion
  <%= augmented_collapsible_section(title: group.to_s) do %>
```

Reply:
And then one question I'd have is whether we could add something as simple as `to_s` to the place that takes the title. 

I.e. in `app/views/augmented/_collapsible_section.html.erb` change:

```diff
- section.with_title { title }
+ section.with_title { title.to_s } 
```

Reply:
It is not `User` but `Category` which I called `.name` for. However, `to_s` also resolves to the name there, so I changed it 👍